src/com/skillsup/view/LoginMenu.java

```diff
@@ -26,85 +28,165 @@ public void show() {
+            String procceding = chooseYesOrNoMenu("Would You like to change user's info? (y/n): ", scanner);
+            if ("y".equalsIgnoreCase(procceding)) {
+                userService.delete(user, choiceUserDB);
+                System.out.println("Input your new login:");
+                String newLogin = scanner.nextLine();
+                System.out.println("Input your new password:");
+                String newPassword = scanner.nextLine();
+                String procced = chooseYesOrNoMenu("Would You like to fill personal info? (y/n) ", scanner);
+                User newUser;
+                if ("y".equalsIgnoreCase(procced)) {
+                    Gender newGender = getGenderInputMenu(scanner);
+                    int newAge = (int) getDoubleInput("Input your new age: ", scanner);
+                    BigDecimal newAccountBalance = BigDecimal.valueOf(getDoubleInput("Input your new account balance: ", scanner));
+                    newUser = new User(newLogin, newPassword, UserRole.CUSTOMER, newGender, newAge, newAccountBalance);
+                } else {
+                    newUser = new User(newLogin, newPassword);
+                }
+                Response<User> registerResponse = userService.register(newUser, choiceUserDB);
+                System.out.println(registerResponse.getFinalMessage());
+            } else System.out.println("Rejection of changes is accepted!");
+        } else {
+            System.out.println(loginResponse.getFinalMessage());
         }
-        return gender;
     }
 
-
-    private void loginSubMenu() {
-        System.out.println("your login:");
+    private void deleteUserSubMenu(String choiceUserDB) {
+        System.out.println("Input your login:");
         String login = scanner.nextLine();
+        System.out.println("Input your password:");
+        String password = scanner.nextLine();
+        Response<User> loginResponse = userService.login(login, password, choiceUserDB);
+        if (loginResponse.isSuccess()) {
+            User user = loginResponse.getData();
+            System.out.println("User logged in :" + user);
+            String procceding = chooseYesOrNoMenu("Would You like to delete this user? (y/n): ", scanner);
+            if ("y".equalsIgnoreCase(procceding)) {
+                System.out.println(userService.delete(user, choiceUserDB).getFinalMessage());
+            } else System.out.println("Rejection of deleting is accepted!");
+        } else {
+            System.out.println(loginResponse.getFinalMessage());
+        }
+    }
 
-        System.out.println("your password:");
+    private void changeAccountBalanceSubMenu(String choiceUserDB) {
+        System.out.println("Input your login:");
+        String login = scanner.nextLine();
+        System.out.println("Input your password:");
         String password = scanner.nextLine();
+        Response<User> loginResponse = userService.login(login, password, choiceUserDB);
```

Comment:
login/pass can be reused, used "session" approach (remembering credentials somewhere in static object)
we dont want user to login again on each operation